### DISCUSSION TITLE
Can't use VFS

### DISCUSSION THREAD

**Discussion by MasterYuan418** [CosmosOS/Cosmos#2387] [Q&A]
I use the code:
```csharp
public static Sys.FileSystem.CosmosVFS fs = new Sys.FileSystem.CosmosVFS();
Sys.FileSystem.VFS.VFSManager.RegisterVFS(fs);
Directory.CreateDirectory("0:\test");
```
I have readed the article: https://cosmosos.github.io/articles/Kernel/VFS.html
It says like "No disks detected.". What should I do?

**Comment by zarlo** [comment]
what are you running this on as you need to have FAT formatted disk's for it to work

**Reply by MasterYuan418** [reply]
I use VMWare. Should I download something like `.vmdk` file?

**Reply by zarlo** [reply]
there should be one that comes with cosmos
